From 9d00e1e860691df416e4919cde485841114c03f3 Mon Sep 17 00:00:00 2001
From: Konstantin Porotchkin <kostap@marvell.com>
Date: Fri, 9 Apr 2021 21:09:21 +0300
Subject: [PATCH 1406/1921] arm64: dts: marvell: fix cn9130-db include file

The cn913-db.dts should include cn9130-db.dtsi, othervise
PCIe interface remains disabled and many other changes from
the base DB include file are missing in final board DTB.

Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Change-Id: I38996d8354db255abf70b039de967e0fbf6b7461
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/49600
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130-db.dts | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-db.dts b/arch/arm64/boot/dts/marvell/cn9130-db.dts
index 33814494a1ba..127647bbd89a 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-db.dts
+++ b/arch/arm64/boot/dts/marvell/cn9130-db.dts
@@ -5,9 +5,7 @@
  * Device tree for the CN9130-DB-A board.
  */
 
-#include "cn9130.dtsi"
-
-#include <dt-bindings/gpio/gpio.h>
+#include "cn9130-db.dtsi"
 
 / {
 	model = "Marvell Armada CN9130-DB setup A";
-- 
2.31.1

